server/src/operators/card_operator.rs

```diff
@@ -545,6 +545,7 @@ pub fn get_metadata_query(
                 user_columns::username,
                 user_columns::website,
                 user_columns::visible_email,
+                user_columns::api_key_hash,
```

Comment:
Do we want their api key in the return request? seems like a security issue

Reply:
I mean the user has to be able to see their created api key some way or another.